### PR TITLE
Home page Design

### PR DESCRIPTION
**Pull request by rahulcs754**
Work Summary

- created note input
image

- created notes slider
image

- created filter Select option
image

- After click filter option
image

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
The code looks clean and well structured,looks good to me

File: src/frontend/Component/Inputarea/Input.module.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.right_footer {
+  flex-basis: 50%;
+  flex-direction: row-reverse;
+  gap: 0.5rem;
+}
+
+@media screen and (max-width: 47.5rem) {
+  .note_input_box {
```
Comment (raman1999) [inline src/frontend/Component/Inputarea/Input.module.css:50]: why u using rem in media query? It's not recommended to do like this, use px

**Review by raman1999 — APPROVED** [review]
code looks good to me , minor suggestion added

File: src/frontend/Component/Inputarea/Input.module.css (hunk 1 above)
Comment (rahulcs754) [reply src/frontend/Component/Inputarea/Input.module.css:50]: yes your are right. i will change next commit

**Pull request merged**